tests/chainerx_tests/unit_tests/routines_tests/test_creation.py

```diff
@@ -259,10 +259,14 @@ def test_array_from_chainerx_array_with_device(
 
     dst_device = chainerx.get_device(dst_device_spec)
 
-    if not copy and src_dtype == dst_dtype and device is dst_device:
+    if not copy and \
```

Comment:
Let's follow the recommendation in PEP-8.
* Use parentheses instead of `\`.
* Place `and` after line breaks.